_posts/platform/app/2000-01-01-log-drain.md

```diff
@@ -32,24 +33,101 @@ Our system is also able to send logs to the following SaaS log processing provid
+* `drain-region`: Used by certain logs management service to identify the region
+of their servers (optional)
+
+The parameters used will depend on the chosen provider.
 
-The format of the environment variable depends on the provider:
+TODO(pc): check if we reference to elk self hosted documentation:
+https://doc.scalingo.com/platform/getting-started/getting-started-with-elk
 
-* Scalingo hosted ELK stack: `https://USER:TOKEN@logstash-app-name.osc-fr1.scalingo.io`
-* Datadog: `datadog://DD_API_KEY?region=REGION`. `DD_API_KEY` is the Datadog API key you can
-    find [here](https://app.datadoghq.com/account/settings#api). `REGION` value
+Below, an exhaustive list of examples of additions for each of the supported
+vendors
+
+* Scalingo hosted ELK stack:
+    ```bash
+    scalingo --app my-app log-drains-add --type elk --url https://my-user:123456789abcdef@logstash-app-name.osc-fr1.scalingo.io
+    ```
+
+* Datadog:
+    ```bash
+    scalingo --app my-app log-drains-add --type datadog --token 123456789abcdef --drain-region eu
+    ```
+    Where
+    * `token` parameter is the Datadog API key you can find
+    [here](https://app.datadoghq.com/account/settings#api).
+    * `region` parameter
     is either `eu` or `us` depending on your Datadog account region.
-* Logentries: `https://webhook.logentries.com/noformat/logs/LOG-TOKEN`.
-    Instructions to get the `LOG-TOKEN` are available on [Logentries
+
+
+* Logentries:
+    ```bash
+    scalingo --app my-app log-drains-add --type logentries --token 123456789abcdef
+    ```
+    Where `token` parameter is the logentries `LOG-TOKEN`, instructions to get
+    it are available on [Logentries
     documentation](https://docs.logentries.com/docs/http-post).
-* OVH hosted Graylog: `ovh://:TOKEN@REGION.logs.ovh.com:6514`. Instructions to
-    get the `TOKEN` and `REGION` variables are available on [OVH
+
+* OVH hosted Graylog:
+    ```bash
+    scalingo --app my-app log-drains-add --type ovh-graylog --token 123456789abcdef --host tag3.logs.ovh.com
+    ```
+    Where
+    - `token` is the graylog token
+    - `host` is the access point provided
+    by ovh. Instructions to get the these variables are available on [OVH
     documentation](https://docs.ovh.com/fr/logs-data-platform/quick-start/).
-* Papertrail: `syslog+tls://logsN.papertrailapp.com:XXXXX`. `logsN` and `XXXXX`
-    are the log host and port number shown under [log
+
+* Papertrail:
+    ```bash
+    scalingo --app my-app log-drains-add --type papertrail --host logs2.papertrailapp.com --port 12345
+    ```
+    `host` and `port` are the log host and port number shown under [log
     destination](https://papertrailapp.com/account/destinations).
-* Syslog TCP+TLS: `tcp+tls://example.com:12345`
+
+* Syslog TCP+TLS:
+    ```bash
+    scalingo --app my-app log-drains-add --type syslog --host custom.logstash.com --port 12345
+    ```
+
+
+
+### List Log Drains
+
+As you can add multiple log drains by application, you can list them with the
+command:
```

Comment:
I don't like the wording here. Suggestion:

```suggestion
You can list all log drains of an applicatino with the `log-drains` command:
```

Reply:
I didn't like it too. Thanks for this suggestion :+1: